### DISCUSSION TITLE
Mob rotation videos

### DISCUSSION THREAD

**Discussion by GitPaulo** [djcooke/compendium#63] [Ideas]
Something that helps is visualising the enemy mob's rotation. It would be really useful if we supported the submission of small videos showing the full rotation for medium/hard-difficulty enemies.

It would sit in enemy details banner:
![image](https://github.com/djcooke/compendium/assets/18451329/a3ff6e62-4f74-45df-af93-268a5aff0b7f)

If so, a few points to consider:
- These videos would be job agnostic and simple - show the mob doing its thing.
- These videos would be optional on the page.
- Force a format: 720p, MP4, Similar Codecs.
- Size limit ~10-50 MB.

@djcooke think it's too much work or not that useful? 🤔

**Comment by djcooke** [comment]
Hey, thanks for the suggestion! I wouldn't want to put these videos in the Github repo, but linking/embedding YouTube would be fine. The videos would be useful, but I don't think they'd be used frequently, so a link or pop-up is probably better to minimize the space used.

Adding these optional videos to the site's design would be easy enough. It's creating/collecting the videos that would be time-consuming. I want to add party and job-specific boss videos as well, but haven't put the time into it.

Let me know if this is something you'd like to work on and I'd be happy to either work with you or just give some more input on the design - whatever you'd prefer

**Comment by GitPaulo** [comment]
Cleaning this up. Re-open if interest exists.
